src/users/users.controller.ts

```diff
@@ -72,21 +73,18 @@ export class UsersController {
   @ApiOperation({ summary: '로그인' })
   @UseGuards(LocalAuthGuard) //인터셉터보다 먼저 실행
   @Post('login')
-  login(@Body() content: LoginReqDto) {
-    return content;
+  async login(@Body() content: LoginReqDto) {
+    const user = this.usersService.getOneUser(content.email);
+    return UsersDto.fromEntity(await user);
```

Comment:
```suggestion
  async login(@InjectUser() user: User) {
    return UsersDto.fromEntity(user);
```

로그인이 성공한 뒤 해당 함수가 실행되니깐, 이렇게 Decorator를 써도 되겠네요~

로그인이 성공해야 일로 오니깐,

Reply:
수정사항 반영했습니다!